common-app-lib/src/cmr/common_app/static.clj

```diff
@@ -184,6 +183,15 @@
        (md->html)
        (selmer/render {})))
 
+(defn parse-out-env [cmr-host]
```

Comment:
Suggested during review that we should just use a `defconfig` in retrospect that does make more sense and simplifies the possibility of errors as well as increases maintainability